evennia/contrib/grid/ingame_map_display/ingame_map_display.py

```diff
@@ -0,0 +1,322 @@
+"""
+Basic Map - helpme 2022
+
+This adds an ascii `map` to a given room which can be viewed with the `map` command.
+You can easily alter it to add special characters, room colors etc.
```

Comment:
It would be relevant to mention already up here that the map is dynamically generated on the fly from the existing grid and that it supports all compass directions + up/down.